src/Models/Device.cs

```diff
@@ -8,13 +8,14 @@ namespace Wangkanai.Detection.Models
     [Flags]
     public enum Device
     {
-        Desktop = 0,      // Windows, Mac, Linux
-        Tablet  = 1 << 0, // iPad, Android
-        Mobile  = 1 << 1, // iPhone, Android
-        Watch   = 1 << 2, // Smart Watchs
-        Tv      = 1 << 3, // Samsung, LG
-        Console = 1 << 4, // XBox, Play Station
-        Car     = 1 << 5, // Ford, Toyota
-        IoT     = 1 << 6  // Raspberry Pi
+        Unknown = 0,
```

Comment:
Using device unknown maybe a cleaner than to have a conditional check if user preference `IsSet` save into session store.